Cargo.lock

```diff
@@ -7451,25 +7626,23 @@ dependencies = [
 
 [[package]]
 name = "moka"
-version = "0.12.8"
+version = "0.12.10"
 source = "registry+https://github.com/rust-lang/crates.io-index"
-checksum = "32cf62eb4dd975d2dde76432fb1075c49e3ee2331cf36f1f8fd4b66550d32b6f"
+checksum = "a9321642ca94a4282428e6ea4af8cc2ca4eac48ac7a6a4ea8f33f76d0ce70926"
 dependencies = [
  "async-lock",
- "async-trait",
  "crossbeam-channel",
  "crossbeam-epoch",
  "crossbeam-utils",
  "event-listener 5.3.1",
  "futures-util",
- "once_cell",
+ "loom",
  "parking_lot 0.12.1",
- "quanta",
```

Comment:
I'm thinking would this break madsim 🤔

Reply:
https://github.com/moka-rs/moka/pull/482/ quanta is replaced by `std::time::Instant` by default. So correctness-wise it should be fine. Performance-wise I guess it's also ok

Reply:
We can remove `quanta` from `[patch.crates-io]`.